packages/main/src/themes/SideNavigation.css

```diff
@@ -0,0 +1,30 @@
+:host(:not([hidden])) {
+    display: inline-block;
+    width: 250px;
+    height: 100%;
+    border-right: var(--sapList_BorderWidth) solid var(--sapList_GroupHeaderBorderColor);
```

Comment:
I don't mind doing RTL at a later stage, but it would be best to use `effectiveDir` and set this for the left too